pom.xml

```diff
@@ -459,7 +461,7 @@
             <dependency>
                 <groupId>com.clickhouse</groupId>
                 <artifactId>clickhouse-jdbc</artifactId>
-                <version>0.6.0-patch4</version>
+                <version>0.6.1</version>
```

Comment:
> Alpha version of the new client API https://github.com/ClickHouse/clickhouse-java/releases/tag/v0.6.1

Is this change unrelated to the connector's usage?

Reply:
I think that it's unrelated (opt-in).